spring-boot/admin/src/main/java/eu/coatrack/admin/security/CustomUserInfoTokenServices.java

```diff
@@ -0,0 +1,51 @@
+package eu.coatrack.admin.security;
+
+/*-
+ * #%L
+ * coatrack-admin
+ * %%
+ * Copyright (C) 2013 - 2022 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.springframework.boot.autoconfigure.security.oauth2.resource.UserInfoTokenServices;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.oauth2.common.OAuth2AccessToken;
+import org.springframework.security.oauth2.common.exceptions.InvalidTokenException;
+import org.springframework.security.oauth2.provider.OAuth2Authentication;
+import org.springframework.security.oauth2.provider.token.ResourceServerTokenServices;
+
+public class CustomUserInfoTokenServices implements ResourceServerTokenServices {
+
+    private final String userInfoEndpointUrl;
+    private final String clientId;
+
+    public CustomUserInfoTokenServices(String userInfoEndpointUrl, String clientId) {
+        this.userInfoEndpointUrl = userInfoEndpointUrl;
+        this.clientId = clientId;
+    }
+
+    @Override
+    public OAuth2Authentication loadAuthentication(String accessToken)
+            throws AuthenticationException, InvalidTokenException {
```

Comment:
Why is there a line break? The line does not seem so long as to exceed the threshold.

Reply:
So this seemed to be because I use my work PC without any extra monitor, and to be able to see the method, I did the line break.
So this is Done